cleverhans/attacks_tf.py

```diff
@@ -57,6 +63,38 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
     return adv_x
 
 
+def fgsm_iterative(x, y, model, eps, eps_iter, n_iter,
```

Comment:
Yes I think the Attack class idea is a good one. Who is planning to take the lead on that?